packages/components/tag/src/tag-list.ts

```diff
@@ -41,14 +40,11 @@ export class TagList extends ScopedElementsMixin(LitElement) {
   /** @internal */
   static override styles: CSSResultGroup = styles;
 
-  /** The maximum width of the +99 stack counter; used for calculating the (in)visible tags. */
-  #maxStackInlineSize = 0;
-
   /**
-   * Observe changes to the size of the tag-list so we can determine when to display
-   * a counter with amount of hidden tags.
+   * Observe size changes so we can determine when to display a counter
+   * with the amount of hidden tags.
    */
-  #resizeObserver = new ResizeObserver(() => this.#updateVisibility());
+  #resizeObserver = new ResizeObserver(entries => this.#onResize(entries));
```

Comment:
I cannot focus any tag in the list, all tags have "tabindex="-1":
https://kind-plant-088739303-1813.westeurope.1.azurestaticapps.net/?path=/story/feedback-status-tag-list--stacked-removable

but I found it only when I go eg. to the story https://kind-plant-088739303-1813.westeurope.1.azurestaticapps.net/?path=/story/feedback-status-tag-list--stacked 
and then go back to this one:
https://kind-plant-088739303-1813.westeurope.1.azurestaticapps.net/?path=/story/feedback-status-tag-list--stacked-removable
so it looks like sth is not being refreshed.

Reply:
The inability to focus is related to resizing the tag-list. We'll create a separate issue for that (#1820)